broker/lib/index.js

```diff
@@ -1,16 +1,15 @@
 'use strict';
 
 exports.jwt = require('./jwt');
-exports.store = require('./store');
 exports.middleware = require('./middleware');
-exports.fabrik = require('./fabrik');
 exports.bootstrap = bootstrap;
 const config = require('../../common/config');
 if (config.enable_swarm_manager) {
   exports.docker = require('../../data-access-layer/docker');
 }
 const logger = require('../../common/logger');
 const docker = exports.docker;
+require('./fabrik/UnlockResourcePoller');
```

Comment:
move this UnlockResourcePoller out of fabrik.

Reply:
put it in common.

Reply:
Done!